<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinWindow</class>
 <widget class="QDialog" name="JoinWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="configPwLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="phonelabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>E-mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PW</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwConfigLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pw Config</string>
   </property>
  </widget>
  <widget class="QLabel" name="joinLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>JOIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="joinBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Join us?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>JoinWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>JoinWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
